docs/sources/datasources/influxdb.md

```diff
@@ -33,6 +33,8 @@ Name        | Description
 `User`      | Name of your database user
 `Password`  | Database user's password
 `HTTP mode` | How to query the database (`GET` or `POST` HTTP verb). The `POST` verb allows heavy queries that would return an error using the `GET` verb. Default is `GET`.
+`Min time interval` | See [detailed explanation]({{< relref "#min-time-interval" >}})
```

Comment:
For min interval as we have a section and it is in both flux and influxDB flavor it seems like a better idea to just link it.